### ISSUE TITLE
A11y_VSCode_Version Control_Screenreader: Screenreader does not read the error message while creating new file or folder with empty name.

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode-docs#3736]
[Check out Accessibility Insights!](https://github.com/microsoft/AzureStorageExplorer/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#MAS1.3.1;#MAS3.3.1;#Win10;#Visual Studio Code;#NVDA;#JAWS; #CAITestPass_May2020;#FTP;#STP;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.488.1 (Official build) dev (64-bit)
Chrome Version: Version 83.0.4103.61 (Official Build) (64-bit)
Screenreader version:NVDA[2020.1], JAWS 2020.2001.70.400
**Repro Steps:**
1.Open Visual studio code editor
2.Using F1 key navigate to the palette, type "git branch" and select "Create Branch"
3. Enter the branch name in the edit field and hit enter. Observe the change in branch name at the bottom left corner
4. Publish branch to the remote Git repository by navigating to the bottom left pane, arriving at the cloud button next to the git branch and hit ENTER
5.Make changes to the code by refactoring, adding/deleting new lines ; using assistive technologies, observe if SRs will alert user the to blue/green/red indicators suggesting if code has been changed/new line added/line has been removed
6.Compare the changes between the HEAD and current file by navigating to View Bar ---> Git ; Navigate to changes and hit ENTER to activate "Diff View" ; use SR to parse code and observe behavior
7. Stage and commit the changes under View Bar ---> Git  ; Navigate to the "Commit Staged"  button and hit ENTER to make a commit
8. Undo the commit by navigating to the toggle button (same row as commit staged), hit ENTER and use down ARROW to choose "Undo Last Commit" option ; Push a commit following the same process
9. Navigate Git commands under Git output under the "toggle button" in step 7
10.Navigate to New folder or New file button and click on that.
**Actual:**
While navigating through keyboard,Screenreader does not read the error message while creating new file or folder with empty name.
This issue is reproduced with NVDA and Jaws.

**Expected:**
Screenreader should read the error message while creating new file or folder with empty name.

**Attachment link for Reference:**

https://microsoft-my.sharepoint.com/:v:/p/v-nisaho/EVHF_7m6NC5MhlIlK9TLRfkBp291YzQ84EzxYffOQ-kCGw?e=AcFFVI

**Comment by gregvanl** [comment]
This looks like an issue with the VS Code product, not the VS Code website at code.visualstudio.com. Product issues should be created in the VS Code product repositiory at https://github.com/microsoft/vscode/issues

**Comment by nirmalsahoo** [comment]
Closing the issue as per incorrect repository.
